libs/host/Configuration/Options.cs

```diff
@@ -493,7 +493,7 @@ public bool IsValid(out List<string> invalidOptions, ILogger logger)
             return isValid;
         }
 
-        public GarnetServerOptions GetServerOptions(ILogger logger = null)
+        public GarnetServerOptions GetServerOptions(IAuthenticationSettings authenticationSettings = null, ILogger logger = null)
```

Comment:
I am not sure I understand why we need this. Instead of passing authenticationSettings as a parameter one could do the following if they need custom authenticationSettings

this.opts = serverSettings.GetServerOptions(this.loggerFactory.CreateLogger("Options"));
this.opts.AuthSettings = new CustomAuthenticationSettings();

Reply:
+1

Reply:
@vazois , Updated.